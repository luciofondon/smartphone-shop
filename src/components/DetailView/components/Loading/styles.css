.loading {
    font-size: 40px;
    text-align: center;
    padding-top: 200px;
}
