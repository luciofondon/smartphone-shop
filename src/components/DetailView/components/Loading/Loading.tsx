import React from 'react';

import './styles.css';

function Loading() {
  return <div className="loading">Cargando...</div>;
}

export default Loading;
