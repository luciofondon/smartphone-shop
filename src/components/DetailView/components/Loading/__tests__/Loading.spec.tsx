import React from 'react';
import { render } from '@testing-library/react';

import Loading from '../Loading';

test('render Loading', () => {
  const { container } = render(<Loading />);
  expect(container).toMatchSnapshot();
});
