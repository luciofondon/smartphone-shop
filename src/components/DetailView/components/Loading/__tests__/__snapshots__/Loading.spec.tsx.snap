// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Loading 1`] = `
<div>
  <div
    class="loading"
  >
    Cargando...
  </div>
</div>
`;
